### ISSUE TITLE
Block Actions: Tooltips unavailable on v16.2

### ISSUE THREAD

**Issue by ppsyrra** [umbraco/Umbraco-CMS#20422]
### Which Umbraco version are you using?

16.2.0

### Bug summary

When hovering over the action icons in the Block Grid Element preview, the tooltip explaining the actions of each icon is missing. 

### Specifics

On v13: 
![WhatsApp Image 2026-02-06 at 14 07 51](https://github.com/user-attachments/assets/0f35048c-8f28-4543-bf9a-8f0c07f89307)

On v17:
<img width="1482" height="187" alt="Screenshot 2026-02-06 135855" src="https://github.com/user-attachments/assets/82d42245-850c-4b56-bc35-5e4b244b1a69" />
### Steps to reproduce

1. Go to Umbraco BO v16.2
2. Open a page with a Block grid
3. Add grid content
4. Hover over the action icons on top right
5. Tooltip is missing

Please see video attached

[Missing tooltips.zip](https://github.com/user-attachments/files/22764352/Missing.tooltips.zip)

### Expected result / actual result

Expected result: Tooltip should be there and explaining the action of the icon

**Comment by iOvergaard** [comment]
Thanks for the issue report, @ppsyrra. This has not previously been implemented in the new Backoffice (14+), but it is certainly something we aim to bring back into the Backoffice.

**Comment by bjarnef** [comment]
It may be something to consider in UI library as alternative to the popover:
https://github.com/umbraco/Umbraco.UI/discussions/1082

**Comment by simonech** [comment]
@iOvergaard any idea on the timeline of this? 17? or later?

/cc @AndyButland

**Comment by nielslyngsoe** [comment]
The uui-button now supports setting a `title` as of (https://github.com/umbraco/Umbraco.UI/pull/1199) so this can now be implemented. It should be a question of declaring the title for the action buttons with the desired title. Do notice from a Accessiblity perspective a title can be more describtive than a button-label. A title can be a spoken out sentence on what the button does. Where the label should be short and concrete.

I will mark this issue as up-for-grabs

**Comment by AndyButland** [comment]
Fixed via https://github.com/umbraco/Umbraco-CMS/pull/21842 for 17.3.

**Comment by AndyButland** [comment]
Re-opening as this was closed in error.  The PR linked above resolve the issue for block list, not block grid.

**Comment by AndyButland** [comment]
Fixed now via https://github.com/umbraco/Umbraco-CMS/pull/21958, for 17.3.
